modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/SingleValueSetter.java

```diff
@@ -45,6 +46,6 @@ public SingleValueSetter(ParameterModel parameterModel, Field field)
     @Override
     public void set(Object target, ResolverSetResult resolverSetResult)
     {
-        fieldSetter.set(target, resolverSetResult.get(parameterModel));
+        Optional.ofNullable(resolverSetResult.get(parameterModel)).ifPresent(v -> fieldSetter.set(target, v));
```

Comment:
but in this case I have a nullable value, and only if its present i want to process it.  
This way I dont assing a variable just for checking the state Optional does it for me